### PR TITLE
Mention maintainer when his module is modified

### PR DESCRIPTION
**Pull request by SimoRubi**
Closing #63

### CODE REVIEW

File: src/oca_github_bot/tasks/mention_maintainer.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+@task()
+@switchable("mention_maintainer")
+def mention_maintainer(org, repo, pr, dry_run=False):
+    with github.login() as gh:
+        gh_pr = gh.pull_request(org, repo, pr)
+        target_branch = gh_pr.base.ref
+        with github.temporary_clone(org, repo, target_branch) as clonedir:
+            cwd = os.getcwd()
+            os.chdir(clonedir)
```
[the remark this answers is not in the document]
Comment (SimoRubi) [reply src/oca_github_bot/tasks/mention_maintainer.py]: yess, that was the general idea! Sorry I missed it, done

File: src/oca_github_bot/tasks/mention_maintainer.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            # Get modified addons list on the PR.
+            pr_branch = f"tmp-pr-{pr}"
+            github._git_call(
+                ["git", "fetch", "origin", f"refs/pull/{pr}/head:{pr_branch}"]
+            )
```
[the remark this answers is not in the document]
Comment (SimoRubi) [reply src/oca_github_bot/tasks/mention_maintainer.py]: done

**Comment by sbidoul** [comment]
@SimoRubi  would you mind rebasing this? There is a conflict (sorry) following a rework of subprocess calls for better error handling and Sentry support. It should be quite easy.

**Comment by SimoRubi** [comment]
> @SimoRubi would you mind rebasing this? There is a conflict (sorry) following a rework of subprocess calls for better error handling and Sentry support. It should be quite easy.

Fixing the mock tests was less easy than expected, anyway here it is and squashed in only one commit

**Comment by sbidoul** [comment]
Great job, thanks!

**Comment by sbidoul** [comment]
deployed

**Pull request merged**